django/db/models/fields/related.py

```diff
@@ -301,6 +301,13 @@ def resolve_related_class(model, related, field):
                 field.remote_field.model = related
                 field.do_related_class(related, model)
             lazy_related_operation(resolve_related_class, cls, self.remote_field.model, field=self)
+        else:
+            # Make sure lazy reference is bound to an application.
```

Comment:
Is it correct to say, "Bind a lazy reference to the app in which the model is defined."
Maybe a helper method is worth avoiding some repetition?